docs/sources/auth/planning/index.md

```diff
@@ -122,6 +122,10 @@ Service account access tokens inherit permissions from the service account.
 
 ### 🚧 API keys
 
+Grafana API keys can be used to interact with data sources via HTTP APIs. API keys can have a well-defined and limited scope to resources with the help of [Roles].
```

Comment:
I'm not sure what "to resources with the help of roles" means.

Reply:
I meant a limited scope or limited access to resources (editing dashboards, viewing sensitive information) with a well-defined role/permission definition.

If this is not clear at first glance, I should reword it.

Reply:
You can only use the basic roles (Viewer, Editor, Admin) for API keys. Service accounts are more flexible and Enterprise users can assign specific RBAC roles to them.